plugin.video.retrospect/resources/libs/chn_class.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, channel_info):
         self.path = channel_info.path
         self.version = channel_info.version
         self.adaptiveAddonSelectable = channel_info.adaptiveAddonSelectable
-        self.hasSettings = channel_info.settings is not None
+        self.hasSettings = channel_info.settings is not None and len(channel_info.settings) > 0
```

Comment:
why not just `self.hasSettings = bool(channel_info.settings)` ?

Reply:
Never know that was possible :)